site: edited installation page and switched Download link to point at it (#603)
kopia/kopia: site/content/docs/Installation/_index.md

weight: 20
---

### Installing Kopia 

> Kopia in an open source software (OSS) developed by a community on GitHub.

Reviewer: ```suggestion
Kopia is an open source software (OSS) developed by a community on GitHub.
```

The recommended way of installing Kopia is to use a package manager for your operating system (YUM or APT for Linux, Homebrew for macOS, Scoop for Windows). They offer quick and convenient way of installing and keeping Kopia up-to-date. See below for more information.

You can also download the [Source Code](https://github.com/kopia/kopia/) or [Binary Releases](https://github.com/kopia/kopia/releases/latest) directly from GitHub. 

Kopia is available in two variants:

* `Command Line Interface (CLI)` which is a stand-alone binary called `kopia` and which can be used a terminal window or scripts. This is typically the preferred option for power users, system administrators, etc.

> * `Graphical User Interface (GUI)` called `KopiaUI` which is a desktop application that offers graphical user interface tailored towards less technical users. It has fewer features than CLI but 

Reviewer: This sentence seems to be incomplete.

Author: removed.

* `Command Line Interface (CLI)` which is a stand-alone binary called `kopia` and which can be used a terminal window or scripts. This is typically the preferred option for power users, system administrators, etc.

> * `Graphical User Interface (GUI)` called `KopiaUI` which is a desktop application that offers graphical user interface tailored towards less technical users.

Reviewer: ```suggestion
* `Graphical User Interface (GUI)`: It is a desktop application, called `KopiaUI`, that offers a friendly user interface.
```
May want to tweak the previous line to match this one.
